source/documentation/how-to/guest-agent/install-the-guest-agent-in-centos.html.md

```diff
@@ -14,7 +14,7 @@ Installing the Guest Agent in a CentOS virtual machine is fairly simple and will
 
 From a terminal session, type the following (for oVirt 4.2)
 
-    sudo yum install centos-release-ovirt42
+    sudo yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm
     sudo yum install ovirt-guest-agent-common
```

Comment:
I would have done:
sudo yum install centos-release-ovirt42, being the guest agent included in CentOS Virt SIG.

Reply:
I agree with @sandrobonazzola 

I have found the root cause just now in the .repo files that where failing. Yum is not able to pick up the value of the $contentdir and by replacing $contentdir with centos. So by fixing the repo files the original content in this agent install document can be left unchanged.

```shell
[root@localhost yum.repos.d]# grep -inr \$contentdir *
CentOS-oVirt-4.2.repo:8:baseurl=http://mirror.centos.org/$contentdir/$releasever/virt/$basearch/ovirt-4.2/
CentOS-QEMU-EV.repo:8:baseurl=http://mirror.centos.org/$contentdir/$releasever/virt/$basearch/kvm-common/
```

#FIX 1
```shell
[root@localhost yum.repos.d]# vi CentOS-oVirt-4.2.repo

#baseurl=http://mirror.centos.org/$contentdir/$releasever/virt/$basearch/ovirt-4.2/
baseurl=http://mirror.centos.org/centos/$releasever/virt/$basearch/ovirt-4.2/
```

#FIX 2 
```shell
[root@localhost yum.repos.d]# vi CentOS-QEMU-EV.repo

#baseurl=http://mirror.centos.org/$contentdir/$releasever/virt/$basearch/kvm-common/
baseurl=http://mirror.centos.org/centos/$releasever/virt/$basearch/kvm-common/
```

[oVirt.guest-agent-error.txt](https://github.com/oVirt/ovirt-site/files/2105217/oVirt.guest-agent-error.txt)

Reply:
yes, better to fix the repo files, so people going through yum search and other ways to find the package would be able to install it. Also the locate might one day need to be changed, so better to avoid hardcoded URLs.

Reply:
@duck-rh Where should I create a issue for fixing the contendir in the centos-release-ovirt42 package ?

Reply:
I think it would be there: https://ovirt-jira.atlassian.net/
Don't forget to check if it was already reported.

Reply:
@samueljon @duck-rh please note this has been already fixed. You need to yum update centos-release on your system since the one shipped in release media is broken.